### ISSUE TITLE
cpecomp_2_2.py _is_valid_edition() always returns true

### ISSUE THREAD

**Issue by MinamiSun** [nilp0inter/cpe#37]
Hi , as the title mentioned,  _is_valid_edition()  in cpecomp_simple.py has a function _is_valid_edition(), which get the value from _is_valid_value() from cpecomp_2_2.py if we use this standard.  _is_valid_value()  returns true or false ,   _is_valid_edition() returns _is_valid_value() is None, which should always be true.

**Comment by nilp0inter** [comment]
You are right. I think we lack some testing here. I'll be doing a new test set and a patch for this in the following days.

Thank you!
